micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -69,7 +70,7 @@ public abstract class MeterRegistry {
     private static final WarnThenDebugLogger doubleRegistrationLogger = new WarnThenDebugLogger(MeterRegistry.class);
 
     // @formatter:off
-    private static final EnumMap<TimeUnit, String> BASE_TIME_UNIT_STRING_CACHE = Arrays.stream(TimeUnit.values())
+    private static final EnumMap<TimeUnit, @NonNull String> BASE_TIME_UNIT_STRING_CACHE = Arrays.stream(TimeUnit.values())
```

Comment:
This was an attempt to get NullAway to understand that `get` on `EnumMap` does not return null, but it doesn't work - hence I had to suppress NullAway on `getBaseTimeUnitStr()`